client/homebrew/editor/editor.jsx

```diff
@@ -191,6 +232,7 @@ const Editor = createClass({
 
 	renderEditor : function(){
 		if(this.isText()){
+			this.props.setMoveArrows(true);
```

Comment:
I think it's generally discouraged to update the state inside of Rendering, since updated state is what triggers a Render. It's possible to end up in an infinite loop this way.

Is it possible to call this inside of the `handleViewChange()` maybe?

Reply:
Just moved the logic to `handleViewChange`, should be in the next commit which will be pushed shortly.